blueocean-rest-impl/src/test/java/io/jenkins/blueocean/service/embedded/PipelineApiTest.java

```diff
@@ -797,13 +799,13 @@ public void parameterizedFreestyleTestWithoutDefaultParam() throws Exception {
                 ), 400);
     }
 
+    @Ignore
```

Comment:
We should be careful keeping such tests in unit tests, as its not really testing much of blueocean. We can always unit test DefaultRunImpl class, that it was ending up testing. There are other compatibility tests that are going to cause false positives like this one.

If at all we want user functional test to make sure maven style project should work with blueocean, these should be added as part of ATH. Perhaps it should be done in that new ticket.